.github/workflows/ci_clang_format_check.yml

```diff
@@ -0,0 +1,17 @@
+name: clang-format Check
+on: [push, pull_request]
+jobs:
+  formatting-check:
+    name: Formatting Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        path:
+          - 'stuffer'
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run clang-format style check for C/C++/Protobuf programs.
```

Comment:
I don't think we'll ever have protobuf

```suggestion
      - name: Run clang-format style check for C
```

Reply:
We actually, surprisingly, do have some cpp code in out code base. -- I'll change this to reflect that.